DSpace/dspace-angular
# In Advanced Search, list collections alphabetically

## References
_Add references/links to any related issues or PRs. These may include:_
* Fixes [#8408](https://github.com/DSpace/DSpace/issues/8408)

## Description
Add alphabetical ordering to community and collections listing.

## Instructions for Reviewers
Add alphabetical ordering to the collection's list of advanced search. For consistency we add ordering to some other collections list.

List of changes in this PR:
The alphabetical ordering affects:
* Advanced search
* Create community and collection
* Edit community, collection and article

To test this PR should be checked:
http://localhost:4000/search -> click "All of Dspace". 
Logged as administrator try to:
create a community
create a collection
edit a community
edit a collection
edit an item

The results should be ordered.
Entering a search text, the results should remain ordered.

## Checklist
